src/bindings/lua/flux-lua.c

```diff
@@ -254,8 +254,11 @@ static int l_flux_new (lua_State *L)
 static int l_flux_kvsdir_new (lua_State *L)
```

Comment:
commit seems to do more than just update to use ```flux_kvs_lookup_get_dir```.  Perhaps commit message should just be enhanced.

Reply:
oh yeah, this specific line is not relevant, just didn't know how to commit on a commit message.

Reply:
Yes that was pretty vague.  Changed to
>   bindings/lua: use flux_kvs_lookup()
> 
>  Replace use of soon to be deprecated flux_kvs_get()
    and flux_kvs_get_dir() with modern flux_kvs_lookup()
    equivalents.